ROADMAP.md

```diff
@@ -8,21 +8,21 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.13
+## Rook Ceph 1.14
 
-The following high level features are targeted for Rook v1.13 (November 2023). For more detailed project tracking see the [v1.13 board](https://github.com/rook/rook/projects/30).
+The following high level features are targeted for Rook v1.14 (April 2024). For more detailed project tracking see the [v1.14 board](https://github.com/rook/rook/projects/31).
 
-* OSD encryption on partitions [#10984](https://github.com/rook/rook/issues/10984)
+* Allow setting the application name on a CephBlockPool [#13744](https://github.com/rook/rook/pull/13744)
```

Comment:
1) add replica-1 support for external clusters
And internal cluster documentation

2) Support for service accounts